src/Stratis.Bitcoin.IntegrationTests/EnvironmentMockUpHelpers/NodeRunners.cs

```diff
@@ -29,6 +32,12 @@ public BitcoinCoreRunner(string bitcoinD)
 
         private Process process;
 
+        public FullNode FullNode
+        {
+            get { return null; }
```

Comment:
`get { return null; }` hu?

Reply:
As part of the refactor I wanted all the runners to implement the interface, but because bitcoin core starts up the bitcoin.exe we don't have a FullNode. I can change this to throw a `NotImplementedException`?

Reply:
ah yeah thats core, yeah throw is better.